Sources/tart/Commands/Get.swift

```diff
@@ -0,0 +1,31 @@
+import ArgumentParser
+import Foundation
+
+struct Get: AsyncParsableCommand {
+  static var configuration = CommandConfiguration(commandName: "get", abstract: "Get a VM's configuration")
+
+  @Argument(help: "VM name")
+  var name: String
+
+  func run() async throws {
+    do {
+      let vmDir = try VMStorageLocal().open(name)
+      let vmConfig = try VMConfig(fromURL: vmDir.configURL)
+      let diskSize = try vmDir.sizeBytes() / 1024 / 1024 / 1000
+
+      print("cpu\tmemory\tdisk\tdisplay")
+
+      var s = "\(vmConfig.cpuCount)\t"
+      s += "\(vmConfig.memorySize / 1024 / 1024)MB\t"
+      s += "\(diskSize)GB\t"
```

Comment:
See https://github.com/cirruslabs/tart/pull/309/files#r1017574000.

```suggestion
      s += "\(diskSize) GiB\t"
```

Reply:
macOS doesn't use `GiB` in the UI even though `GB` is `GiB`. See https://github.com/cirruslabs/tart/blob/8961c5189a792cda828b8e43dade23c38bae5736/Sources/tart/VMDirectory.swift#L84-L85

So to be consistent with macOS Finder we should IMO stick with GB.